packages/core/src/lib/androidSdk/AndroidSdkTools.ts

```diff
@@ -194,4 +205,19 @@ export class AndroidSdkTools {
     ];
     await util.execute(installCmd, env, this.log);
   }
+
+  /**
+   * Checks if the androidSdkPath in the config file is valid.
+   * @param {Config} config the bubblewrap general configuration.
+   */
+  static async validatePath(config: Config): Promise<Result<boolean, Error>> {
+    const androidSdkPath = config.androidSdkPath;
+    // Checks if the path given is valid.
+    if (!fs.existsSync(path.join(androidSdkPath, 'tools'))|| !fs.existsSync(androidSdkPath)) {
+      return Result.error(new Error('androidSdkPathIsNotCorrect'));
+    };
+    return Result.ok(true);
+  }
 }
+export const newAndroidSdkTools = AndroidSdkTools.newAndroidSdkTools;
```

Comment:
There's not need to export those. They can be accessed via `AndroidSdkTools.validatePath()` / `AndroidSdkTools. newAndroidSdkTools()`

Reply:
Also, let's rename the method to `create`, since `newAndroidSdkTools` becomes redundant after tha.

Reply:
@chenlevy24 this and other items are marked as resolved, but the code is unchanged. Are we possibly missing a commit?